packages/react/src/components/Tree/TreeItem.tsx

```diff
@@ -52,21 +63,22 @@ export interface TreeItemState {
   open?: boolean
 }
 
-class TreeItem extends AutoControlledComponent<ReactProps<TreeItemProps>, TreeItemState> {
+class TreeItem extends UIComponent<ReactProps<TreeItemProps>, TreeItemState> {
   static create: Function
 
   static className = 'ui-tree__item'
 
   static displayName = 'TreeItem'
 
-  static autoControlledProps = ['open']
-
   static propTypes = {
     ...commonPropTypes.createCommon({
       content: false,
     }),
     defaultOpen: PropTypes.bool,
     items: customPropTypes.collectionShorthand,
+    index: PropTypes.number,
+    exclusive: PropTypes.bool,
+    onOpenChanged: PropTypes.func,
```

Comment:
```suggestion
    onOpenChange: PropTypes.func,
```

We should go with `onOpenChange` as it was defined in other components.